msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting.md

```diff
@@ -53,6 +53,100 @@ The following table provides the user types and lists the features that each use
+
+To send targeted in-meeting notifications, you must configure the `authorization` property and the `name` and `type` properties under the `resourceSpecific` field in the [app manifest](../resources/schema/manifest-schema.md#authorization) as follows:
+
+```json
+"authorization": {
+    "permissions": {
+      "orgWide": [],
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeetingNotification.Send.Chat",
+          "type": "Application"
+        }
+      ]
+    }
+  }
+```
+
+### Enable targeted in-meeting notification
+
+> [!NOTE]
+> You can send a targeted in-meeting notification to 50 participants in a meeting. If you want to send a targeted in-meeting notification to more than 50 participants, you must call the `targetedMeetingNotification` API again.
+
+Targeted in-meeting notification can be triggered by user action.
+
+To enable the targeted in-meeting notification:
+
+1. Retrieve the user IDs of the participants through [Get participant API](meeting-apps-apis.md#get-participant-api) and [Get members API](../bots/how-to/get-teams-context.md#fetch-the-roster-or-user-profile).
+
+   > [!NOTE]
+   > Targeted in-meeting notification doesn't support user IDs in the Azure Active Directory (Azure AD) user ID format.
+
+   **The following is an example of a user ID**:
+    `id=29:1I12M_iy2wTa97T6LbjTh4rJCWrtw2PZ3lxpD3yFv8j2YPnweY2lpCPPAn3RIOPP7rghfHauUz48I1t7ANhj4CA`
+
+1. Include the user IDs in the request parameter.
+
+   The following is an example of a request:
+
+    ```http
+     POST /v1/meetings/{meetingId}/notification
+    ```
+
+   The following is an example of a payload:
+
+    ```json
+    {
+      "type": "targetedMeetingNotification",
+      "value": {
+        "recipients": [ 
+        "29:1I12M_iy2wTa97T6LbjTh4rJCWrtw2PZ3lxpD3yFv8j2YPnweY2lpCPPAn3RI0PP7rghfHauUz48I1t7ANhj4CA"
+         ], 
+        "surfaces": [ 
+          { 
+            "surface": "meetingStage", 
+            "contentType": "task", 
+            "content": { 
+              "value": { 
+                "height": "300", 
+                "width": "400", 
+                "title": "Targeted meeting Notification", 
+                "url": "https://somevalidurl.com"           
+                }
+            } 
+          } 
+        ] 
+      },
+      "channelData": { // optional if a developer doesn't want to support user attributes.
+        "onBehalfOf": [ 
+          { 
+            "itemid": 0, 
+            "mentionType": "person", 
+            "mri": "29:1mDOCfGM9825lMHlwP8NjIVMJeQAbN-ojYBT5VzQfPpnst1IFQeYB1QXC8Zupn2RhgfLIW27HmynQk-4bdx_YhA", 
+            "displayName": "yunny chung"      } 
+        ] 
+      }
+    }
+    ```
+
+     Targeted in-meeting notification is enabled.
+
+For more information on `targetedMeetingNotification` API, see [Targeted in-meeting notification API](meeting-apps-apis.md#targeted-in-meeting-notification-api).
+
 ## Code sample
 
 Sample name | Description | .NET | Node.js |
```

Comment:
Code sample page says 'content bubble', and I believe this is an old name for 'in-meeting notification'. Is it possible to update the code sample page as well? We can update it as a separate PR, but in that case, let's have a WI :)

Reply:
content bubble is shown in the link, not sure if that can be removed.